src/sentry/static/sentry/app/views/performance/onboarding.tsx

```diff
@@ -1,28 +1,58 @@
 import React from 'react';
+import styled from '@emotion/styled';
 
 import OnboardingPanel from 'app/components/onboardingPanel';
 import Button from 'app/components/button';
-import {IconLightning} from 'app/icons';
+import space from 'app/styles/space';
 import {t} from 'app/locale';
 
+import emptyStateImg from '../../../images/spot/performance-empty-state.svg';
+
 function Onboarding() {
   return (
-    <OnboardingPanel image={<IconLightning size="200px" />}>
-      <h3>{t('No transactions yet')}</h3>
+    <OnboardingPanel image={<PerfImage src={emptyStateImg} />}>
+      <h3>{t('Pinpoint problems')}</h3>
       <p>
         {t(
-          'View transactions sorted by slowest duration time, related issues, and number of users having a slow experience in one consolidated view. Trace those 10-second page loads to poor-performing API calls and its children.'
+          "You've got this souped up plan. Now what? Get your software set up. We’ve got transactions to track down."
```

Comment:
You have a mix of ascii quotes and apostrophes here, can we normalize on one of them?